soh/src/code/z_actor.c

```diff
@@ -3071,6 +3072,13 @@ void func_80031C3C(ActorContext* actorCtx, PlayState* play) {
         actorCtx->absoluteSpace = NULL;
     }
 
+    if (CVarGetInteger("gStairs", 0)) {
```

Comment:
It would make more sense to me to have all these simulated heap functions controlled by the gSimulateHeap cvar, and then use your gStairs one ONLY for the graveyard use case.

Reply:
so the "gStairs" cvar is just a silent variable (i.e. the user has no control over it) and then gSimulateHeap cvar will set this on/off at the start of play init. this is so that the heap is not allocated memory when it shouldnt, and that the memory always gets freed properly at the end of the play loop

Reply:
I understand the reasoning here, however, if this is going to be useful for emulating more than the Royal Tomb Grave unloading glitch, this variable should have a better name.